VCEntities/VCEntities/verifiableCredential/VCClaims.swift

```diff
@@ -5,13 +5,19 @@
 
 import VCToken
 
+/**
+ * Contents of a Verifiable Credential Token.
+ *
+ * @see [Verifiable Credential Spec](https://www.w3.org/TR/vc-data-model/)
+ */
 public struct VCClaims: Claims {
-    public let jti: String
-    public let iss: String
-    public let sub: String
-    public let iat: Double
-    public let exp: Double
-    public let vc: VerifiableCredentialDescriptor
+    public let jti: String?
+    public let iss: String?
+    public let sub: String?
+    public let iat: Double?
+    public let exp: Double?
```

Comment:
Should we add a test to make sure that we don't throw serialization exception when these properties are missing (for future regressions)

Reply:
yes adding to next pr